blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -53,9 +53,14 @@ Breaking Changes
 Changes
 =======
 
+- Add support for the filter clause in
+  :ref:`window function calls <window-function-call>`.
+  Only window functions that are :ref:`aggregates <aggregation>` can be used
+  with the filter clause.
+
 - Added :ref:`TIMEZONE <scalar-timezone>` scalar function.
 
-- Added support for the filter clause in the
+- Added support for the filter clause in
```

Comment:
Maybe the two `filter` related entries could be merged into one.

Reply:
It is really related implementation wise, but i am not sure that we shall mix the filter clause in aggregate expressions and window functions. Imho it is clearer to keep it as separate change-log entries, but no strong opinion here..

Reply:
I think the filter support is part of the aggregation support and not of the window functions. From a spec perspective, all aggregations can be used as window functions. If aggregations support filters, it is also supported when used by a window functions. That we treated this a bit different implementation wise is imho not relevant from a user perspective. So +1 for merging the entries ;-)

Reply:
ok, will merge